src/src/EmailBundle/Command/SendRegistrationOpenCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace EmailBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use AdminBundle\Entity\Event;
+use EmailBundle\Controller\EmailController;
+
+class SendRegistrationOpenCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName('email:send:registration-open')
+             ->setDescription('Sends notification emails to all subscribed attendees that the registration is open.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $controller = new EmailController();
+        $controller->setContainer($this->getContainer());
+
+        $em = $this->getContainer()->get('doctrine')->getManager();
+        $events = $em->getRepository(Event::class)->getTodaysOpenEvents();
+        $attendees = $em->getRepository(Attendee:class)->getSubscribedAttendees();
+        foreach ($events as $event) {
+            $this->sendNotificationForEvent($event, $attendees, $controller);
+        }
+    }
+
+    private function sendNotificationForEvent($event, $attendees, $controller)
+    {
+        echo 'For address '.$event->getAddress()."\n";
+        echo "--------------------\n";
+
+        $registrations = $event->getRegistrations();
+        foreach ($attendees as $attendee) {
+            $context = $this->getContextForNotificationEmail($event, $attendee, $controller);
+
+            $controller->sendEmail($attendee, $context, $event->getTemplateOverride(), 'new_event');
```

Comment:
tu este treba ocekovat v `email_logu` ci uz mu nahodou ten email nebol raz poslany. takto mu posle 10x to iste, to nechceme.

Reply:
V email logu je successful status reprezentovany cim?

Reply:
A netreba to riesit v tej metode sendEmail? Lebo mimo toho, ze vytvorim ten email kompletne cely nemam ako zistit ci uz existuje.

Reply:
no, je mozne, ze by sa to mohlo riesit v `sendEmail`, riesme to teda tam. a je to reprezentavane tym, ze `status` v `EmailLog` je `0`